WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -59,15 +60,25 @@ struct ItemListView: View {
         VStack(spacing: 0) {
             headerView
 
-            switch itemListState {
-            case .loading(let items),
-                    .loaded(let items, _),
-                    .inlineError(let items, _, _):
-                listView(items)
-            case .error(let errorState):
-                errorView(errorState)
-            case .empty:
-                emptyView
+            if isSearchFieldFocused && searchTerm.isEmpty {
+                POSRecentSearchesView(
```

Comment:
Nice that we can use a different recent searches view, but continue using the existing list for the results.